models/issue_comment.go

```diff
@@ -247,14 +247,20 @@ func (c *Comment) LoadAssignees() error {
 	if c.OldAssigneeID > 0 {
 		c.OldAssignee, err = getUserByID(x, c.OldAssigneeID)
 		if err != nil {
-			return err
+			if !IsErrUserNotExist(err) {
+				return err
+			}
+			c.OldAssignee = NewGhostUser()
```

Comment:
Why not unassign the issue? 🤔

Reply:
Issue is unassigned afaik, this is just comment that issue was assigned to user